### ISSUE TITLE
Usage with SwiftUI

### ISSUE THREAD

**Issue by Elektronenhirn108** [kishikawakatsumi/KeychainAccess#537]
Hello,

does anyone uses the package with SwiftUI? 
Everything works so far, but changes are not fetched in the background. 
I have a list that contains the values from the allKeys()-methods.  So if I add a new password, the list doesn't update because the new password isn't fetched. Same for cloud-sync. If there is a new key in the cloud, I have to call the allKeys()-method manually to fetch changes.
How can I do that automatically?

Help would be great :)

**Comment by Andrew-Chen-Wang** [comment]
have you resolved this?

**Comment by Elektronenhirn108** [comment]
> have you resolved this?

No. As far as I understand is the Keychain not capable of automatic background updates/ fetching. You have to do it manually.
